agent/lib/pbench/tests/fixtures/config/pbench-agent.cfg

```diff
@@ -0,0 +1,57 @@
+[DEFAULT]
+pbench_install_dir = /opt/pbench-agent
+pbench_results_redirector = pbench.results.example.com
+pbench_web_server = pbench.example.com
+
+version = 002
+
+[config]
+path = %(pbench_install_dir)s/config
+files = pbench-agent-default.cfg
```

Comment:
Just include lines 1-10 in your specific config file but change the pbench_install_dir somehow to reflect the place where the tests are run.

Reply:
So the pbench_install_dir should be changed to a path *relative to wherever the pbench-agent.cfg is*.
E.g. suppose I'm sitting at  the top level (where .travis.yml is), the pbench_agent_default.cfg file is `./agent/config/pbench_agent_default.cfg` relative to me and the pbench agent config file is `./agent/lib/pbench/tests/fixtures/config/pbench_agent.cfg` relative to me. Then you should set

     pbench_install_dir = ../../../../..

i.e. go back up to the ./agent directory five levels up before going down to the `%(pbench_install_dir)s/config` subdir to find the default config file.

You should be able to test this with:

     ./agent/util-scritps/getconf.py -L

which should give a list of the two relevant config files with full pathnames.